core-bundle/src/Resources/contao/dca/tl_undo.php

```diff
@@ -113,6 +113,10 @@
 		(
 			'search'                  => true,
 			'sql'                     => "mediumblob NULL"
+		),
+		'preview' => array
+		(
+			'sql'					  => "text NULL"
```

Comment:
Should we may be use a `mediumtext` or  `longtext` here? The complete preview output could in theory be rather long.